Input/Core/InputEventQueue.cs

```diff
@@ -8,7 +8,7 @@ class InputEventQueue
 	{
 		#region Fields
 
-		readonly SortedList<float, InputEvent> m_List = new SortedList<float, InputEvent>(new SortInputEventsByTime());
+		readonly List<InputEvent> m_List = new List<InputEvent>();
```

Comment:
In case it wasn't clear, this changed because SortedList allocates when you add to it